tests/fiaas_deploy_daemon/conftest.py

```diff
@@ -191,11 +191,11 @@ def _mock_namespace_file_open(name, *args, **kwargs):
 
 @pytest.fixture(scope="session", params=(
     "v1.15.12",
-    "v1.16.13",
-    "v1.18.6",
-    "v1.19.4",
+    "v1.16.15",
+    "v1.18.19",
+    "v1.19.11",
     "v1.20.7",
-    pytest.param("v1.21.1", marks=pytest.mark.e2e_latest)
+    pytest.param("v1.21.2", marks=pytest.mark.e2e_latest)
```

Comment:
Is there a reason for bundling changes in these versions together with the switch?

Reply:
this just has to do with the docker images available in official kind versus bsycorp's kind